sql/core/src/test/scala/org/apache/spark/sql/connector/AlterTableTests.scala

```diff
@@ -407,6 +408,65 @@ trait AlterTableTests extends SharedSparkSession {
     }
   }
 
+  test("SPARK-36381: Alter add column exist check in case sensitive") {
```

Comment:
maybe move this to https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/connector/V2CommandsCaseSensitivitySuite.scala?

Reply:
+1

Reply:
Done. Thank you very much @imback82 @cloud-fan